src/ostree/ot-builtin-pull.c

```diff
@@ -57,6 +58,7 @@ static GOptionEntry options[] = {
    { "mirror", 0, 0, G_OPTION_ARG_NONE, &opt_mirror, "Write refs suitable for a mirror and fetches all refs if none provided", NULL },
    { "subpath", 0, 0, G_OPTION_ARG_FILENAME_ARRAY, &opt_subpaths, "Only pull the provided subpath(s)", NULL },
    { "untrusted", 0, 0, G_OPTION_ARG_NONE, &opt_untrusted, "Do not verify checksums of local sources (always enabled for HTTP pulls)", NULL },
```

Comment:
The symmetry between the two options made me realize the description for this is wrong. It should be "**Do** verify...".

Reply:
Ooh yes, good catch!  Rebased 🏄 and added a prep commit for this ⬇️